src/components/titleBar.vue

```diff
@@ -24,19 +24,19 @@
         <div class="download-group">
           <div class="download-info">Available for macOS, Windows and Linux.</div>
           <div class="button-group">
-            <a class="button" href="https://github.com/marktext/marktext/releases/download/v0.15.1/marktext-0.15.1.dmg" target="_blank">
+            <a class="button" href="https://github.com/marktext/marktext/releases/download/latest/marktext.dmg" target="_blank">
```

Comment:
@Lisiadito According to [this](https://help.github.com/en/github/administering-a-repository/linking-to-releases), the correct URL is `https://github.com/marktext/marktext/releases/latest/download/[...]`

Reply:
true... will fix it.